INSTALL.md

```diff
@@ -31,6 +31,8 @@ CONFIG_HAVE_BPF_JIT=y
 CONFIG_BPF_EVENTS=y
 ```
 
+For Linux kernel version 4.7 or newer, the flag `CONFIG_HAVE_BPF_JIT` has been replaced by `CONFIG_HAVE_EBPF_JIT` (for x86, arm64 and s390) or by `CONFIG_HAVE_CBPF_JIT` (for arm, mips, powerpc and sparc.)
```

Comment:
In two cases the information is useful:

1) when users are trying bcc on any Linux distribution, they may check the distribution's pre-installed kernel's compile config to ensure BPF is supported by the kernel. Especially, they could know which kind of BPF (cBPF or eBPF) is supported, which may determine whether some new BPF features are available or not. 

2) when users want to build their own Linux kernel with fully BPF support, these are essential flags that they have to enable in the .config file. 

The second case is exactly the situation I met: 

I was compiling the newest kernel for some bcc experiments. In the kernel's default config for some hardware, BPF isn't fully enabled (e.g., `BPF_SYSCALL` is not set). Hence I checked all flags listed in this `INSTALL.md` document and enabled them one by one. However, I couldn't find the `CONFIG_HAVE_BPF_JIT` option in newest kernel source and was pretty confused. Finally realize the flag was deprecated.